desktop/apps/auction2/components/DOM.js

```diff
@@ -10,19 +10,19 @@ class DOM extends Component {
   static propTypes = {
     auction: PropTypes.object.isRequired,
     children: PropTypes.node.isRequired,
+    infiniteScroll: PropTypes.func.isRequired,
     user: PropTypes.object
   }
 
-  static contextTypes = {
-    store: PropTypes.object.isRequired
-  }
-
   // Selectors
   $ = null
   $body = null
   $registerBtn = null
 
   componentDidMount () {
+    const FastClick = require('fastclick')
+    FastClick(document.body)
```

Comment:
@sweir27 and @erikdstock - When building responsive views make sure to use this; it removes the weird 300ms delay from clicks / taps